src/convertToObject.js

```diff
@@ -13,6 +13,20 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const stylesObj = {};
+
+  const splitStyles = sourceString
+    .split(';')
+    .map(style => style
+      .split(':')
+      .map(item => item.trim()))
+    .filter(item => item.length > 1);
```

Comment:
```suggestion
  const splitStyles = sourceString
    .split(';')
    .map(style => style
      .split(':')
      .map(item => item.trim())
    ).filter(item => item.length > 1);
```
так будет чуть-чуть легче читать